### PR TITLE
Update model.py

### PR DESCRIPTION
**Pull request by SiqiTao**

### CODE REVIEW

File: src/model.py [hunk 1]
```diff
@@ -78,36 +78,27 @@ def main(input, output):
...
+    save_df(result_df, "score_results", output)
 
-    # Creating the best model
-    model = LogisticRegression(
-        max_iter=2000,
-        C=random_search.best_params_["logisticregression__C"]
-    )
-
-    # Storing optimized model
-    pickle.dump(model, open(f"{output}final_model.rds", "wb"))
```
Comment (iamMoid) [inline src/model.py:107]: I think you will need to retain the above script to create the 'model'

Comment (iamMoid) [reply src/model.py:107]: delete the pickle.dump line though. Keep the one below this.

Comment (SiqiTao) [reply src/model.py:107]: its in line 100

**Review by iamMoid — APPROVED** [review]

**Comment by iamMoid** [comment]
Thanks for updating the model.py to the latest code.

**Comment by SiqiTao** [comment]
> Thanks for updating the model.py to the latest code.

thank you for your help sir

**Pull request merged**